.github/workflows/ci_jruby.yml

```
@@ -71,3 +71,51 @@ jobs:
       - name: Annotate errors
         if: ${{ failure() }}
         uses: ./.github/actions/annotate
+
+  jruby_sidekiq:
+    runs-on: ubuntu-22.04
+    services:
+      redis:
+        image: redis
+        ports:
+          - 6379:6379
+        options: >-
+          --health-cmd "redis-cli ping"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    strategy:
+      fail-fast: false
+      matrix:
+        suite: [sidekiq]
+
+    steps:
+      - name: Configure git
+        run: 'git config --global init.defaultBranch main'
+
+      - name: Check out the source code
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # tag v4.2.2
+
+      - name: Install JRuby
+        uses: ruby/setup-ruby@bbda85882f33075a3727c01e3c8d0de0be6146ce # tag v1.224.0
+        with:
+          ruby-version: jruby-9.4.11.0
```

If we decide to bump the other two in #3101, we'll want to do so here too:
```suggestion
          ruby-version: jruby-9.4.12.0
```

---

Lets do it!